locales/it_IT/pcsx2_Iconized.po

```diff
@@ -717,22 +715,25 @@ msgstr ""
 "non corrette. Se hai problemi di emulazione, per prima cosa disattiva le "
 "opzioni in questo pannello."
 
-#: pcsx2/gui/Panels/SpeedhacksPanel.cpp:126
-#, fuzzy
+#: pcsx2/gui/Panels/SpeedhacksPanel.cpp:135
 msgid ""
-"Setting higher values on this slider effectively reduces the clock speed of "
+"Setting lower values on this slider effectively reduces the clock speed of "
```

Comment:
Impostando questo slider su valori più bassi si riduce propriamente la frequenza
del core CPU R5900 dell'EmotionEngine, permettendo in genere di migliorare
l'emulazione di quei giochi che non fanno completo uso di tutto l'hardware PS2. 
Al contrario, valori più alti aumenteranno a tutti gli effetti questa frequenza
portando un aumento degli FPS all'interno dei giochi, al costo di requisiti di
emulazione più esigenti e possibili glitch.